api/datastore/sql/sql.go

```diff
@@ -68,7 +76,7 @@ var tables = [...]string{`CREATE TABLE IF NOT EXISTS routes (
 }
 
 const (
-	routeSelector = `SELECT app_name, path, image, format, memory, type, timeout, idle_timeout, headers, config FROM routes`
```

Comment:
I think for audit purposes we need to add more than just `created_at`, what about `updated_at`, `delete_at`?

Reply:
yes, I agree -- read parent comment. this patch is about adding migrations, not adding timestamps. one was added so as to be an example so that others can do future ones. it's on my short list but don't want to bloat this patch.

Reply:
ok, I'm fine with adding them later, just wanted to clarify if that would happen "soon".